.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Pull requests into this repository require the following checks to be completed by the submitter and reviewers.
+
+**Submitter:**
+- [ ] Administrative tasks are complete.
+  - This pull request describes why these changes were made.
+  - The JIRA ticket links to this PR.
+  - The JIRA ticket for this PR is: 
+- [ ] Tests are complete.
+  - Tests are included and test edge cases.
+  - Tests have been run locally and pass.
+- [ ] Code review has been performed.
+  - Code diff has been done and checked to ensure only expected code is being committed.
+
+**Primary Reviewer:**
+
+- [ ] Code works.
+  - Code has been run locally and the basic functionality affected by the PR has been checked.
+  - Edge cases have been checked to ensure behavior is as expected.
+- [ ] Code review has been performed.
+  - Code is maintainable, reusable, efficient, and correct.
+  - Code accomplishes the tasks purpose.
+  - Code follows style guidance appropriate for the language.
```

Comment:
Should running the linter be explicitly stated somewhere or does it just fall under style guidance?